react/features/conference/components/Conference.web.js

```diff
@@ -94,6 +120,16 @@ class Conference extends Component {
             </div>
         );
     }
+
+    /**
+     * Displays the toolbar.
+     *
+     * @private
+     * @returns {void}
+     */
+    _onShowToolbar() {
+        APP.UI.showToolbar();
```

Comment:
APP.UI.showToolbar seem to already call an action: "UI.showToolbar = timeout => APP.store.dispatch(showToolbox(timeout));". Can't we just dispatch the action directly?